public/app/features/templating/template_srv.ts

```diff
@@ -249,8 +249,9 @@ export class TemplateSrv implements BaseTemplateSrv {
   ): string {
     // Scenes compatability (primary method) is via SceneObject inside scopedVars. This way we get a much more accurate "local" scope for the evaluation
     if (scopedVars && scopedVars.__sceneObject) {
+      const sceneObject = (scopedVars.__sceneObject.value as SafeSerializableSceneObject).valueOf();
```

Comment:
@dprokop don't we need a fallback here for older (ie all existing) scene apps that have not upgraded to scenes 
5.6?